namespace cpp2 facebook.memcache.cpp2

typedef binary (cpp2.type = "folly::IOBuf") IOBuf

struct ManyFields {
  1: IOBuf buf1;
  2: IOBuf buf2;
  3: IOBuf buf3;
  4: IOBuf buf4;
  5: IOBuf buf5;
  6: IOBuf buf6;
  7: IOBuf buf7;
  8: IOBuf buf8;
  9: IOBuf buf9;
  10: IOBuf buf10;
  11: IOBuf buf11;
  12: IOBuf buf12;
  13: IOBuf buf13;
  14: IOBuf buf14;
  15: IOBuf buf15;
  16: IOBuf buf16;
  17: IOBuf buf17;
  18: IOBuf buf18;
  19: IOBuf buf19;
  20: IOBuf buf20;
  21: IOBuf buf21;
  22: IOBuf buf22;
  23: IOBuf buf23;
  24: IOBuf buf24;
  25: IOBuf buf25;
  26: IOBuf buf26;
  27: IOBuf buf27;
  28: IOBuf buf28;
  29: IOBuf buf29;
  30: IOBuf buf30;
  31: IOBuf buf31;
  32: IOBuf buf32;
  33: IOBuf buf33;
  34: IOBuf buf34;
  35: IOBuf buf35;
  36: IOBuf buf36;
  37: IOBuf buf37;
  38: IOBuf buf38;
  39: IOBuf buf39;
  40: IOBuf buf40;
}
